example/styles/vars.scss

```
@@ -0,0 +1,578 @@
+@import './methods/merge-multiple';
```

```suggestion
// DO NOT EDIT: THIS FILE WAS DUPED FROM THE README MONOREPO

@import './methods/merge-multiple';
```

or something to that effect?